.gitignore

```diff
@@ -30,6 +30,7 @@ yarn-error.log*
 
 # vercel
 .vercel
+.vscode
```

Comment:
should be standard in out .gitignore?

Reply:
missed this ... we need to whitelist some files. will demo in upcoming PR.